src/Resources/skeleton/resetPassword/twig_check_email.tpl.php

```diff
@@ -0,0 +1,8 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Password Reset Email Sent{% endblock %}
+
+{% block body %}
+    <p>An email has been sent that contains a link you can click to reset your password. This link will expire in {{ tokenLifetime|date('g') }} hour.</p>
```

Comment:
> ... a link *that* you can click...

Reply:
Also, I guess we should say "hour(s)" on the end. I don't think we should try to introduce the translator so that we can "translate" hour versus hours. Or we could add the proper logic, though it's a little ugly:

```twig
{{ tokenLifetime|date('g') }} hour{{ tokenLifetime|date('g') != 1 ? 's' }} hour
```

If we did that, I might also pass `tokenLifetimeHours` into the template for convenience.

Reply:
I went with `hours(s)`.

From a technical stand point, using the proper logic makes sense. But, I suspect most people will be changing the format of this template to suite their needs (match UI/business themes, etc..). So perhaps we would introduce the "proper logic" at a later time if it is requested..

Reply:
done